events/README.md

```diff
@@ -0,0 +1,472 @@
+
+## GET - /sessions
+
+To retrieve all the sessions.
+
+- Params:
+  - hits: Number of sessions in one response
+  - offset: Start of response
+    - ex: For 1-10 hits: 10, offset: 0; 11-20 hits: 10, offset: 10;
+- Query
+  - None
+- Body
+  - None
+- Headers
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```jsx
+      {
+      	"limit": 10,
+      	"data": [
+      		<session_object>,
+      		<session_object>,
+      	]
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't get sessions. Please try again later'`
+
+## GET - /sessions/:status
+
+To get currently running sessions.
+
+- Params:
+  - active: true/false
+  - hits: Number of sessions in one response
+  - offset: Start of response
+    - ex: For 1-10 hits: 10, offset: 0; 11-20 hits: 10, offset: 10;
+- Query
+  - None
+- Body
+  - None
+- Headers
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```jsx
+      {
+      	"limit": 10,
+      	"data": [
+      		<session_object>,
+      		<session_object>,
+      	]
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't get sessions. Please try again later'`
+
+## GET - /sessions/:id=<SESSION_ID>
+
+Retrieves the details of a specific active session.
+
+- **Params:**
+  - **`id=[string]`** (The ID of the session to retrieve)
+- **Query**
+  - None
+- **Body**
+  - None
+- **Headers**
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "id": "<session_id>",
+        "event_id": "<event_id>",
+        "customer_id": "<customer_id>",
+        "active": false,
+        "peers": [
+      		<peer_object>,
+      		<peer_object>
+      	]
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      **`{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`**
+  - **Code:** 404
+    - **Content**
+      **`{ 'statusCode': 404, 'error': 'Not Found', 'message': 'Session not found' }`**
+  - **Code:** 500
+    - **Content**
+      **`{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Unable to retrieve session details' }`**
```

Comment:
can we add 400 here.

Reply:
We have removed the session APIs since we found out that we won't use it.